packages/wallet/scripts/validate-env.js

```diff
@@ -0,0 +1,73 @@
+const path = require("path");
+const dotenv = require("dotenv");
+const fs = require("fs");
+const difference = require("lodash/difference");
+
+const ENV_PATH = path.join(__dirname, "../.env");
+const ENV_TEMPLATE_PATH = path.join(__dirname, "../.env.template");
+
+function envDiff(arr1, arr2) {
+  const items1 = difference(arr2, arr1);
+  const items2 = difference(arr1, arr2);
+
+  return [items1, items2];
+}
+
+function readEnv(path) {
+  const rawEnvs = fs.readFileSync(path, "utf8");
+
+  return dotenv.parse(rawEnvs);
+}
+
+function validateEnv(platform) {
+  if (!platform) {
+    console.error("Please specify platform value like android or ios");
+    process.exit(1);
+  }
+
+  let envs = Object.keys(readEnv(ENV_PATH));
+  let templateEnvs = Object.keys(readEnv(ENV_TEMPLATE_PATH));
+  const excludeEnvPrefix = platform === "ios" ? "NF_ANDROID" : "NF_IOS";
+
+  // Remove some variables from template envs depending on platform
+  templateEnvs = templateEnvs.filter(e => !e.startsWith(excludeEnvPrefix));
+  envs = envs.filter(e => !e.startsWith(excludeEnvPrefix));
```

Comment:
I think what Pawel meant was

```suggestion
  const excludeEnvPrefix = platform === "ios" ? "NF_ANDROID" : "NF_IOS";
  const envs = Object.keys(readEnv(ENV_PATH)).filter(e => !e.startsWith(excludeEnvPrefix));;
  const templateEnvs = Object.keys(readEnv(ENV_TEMPLATE_PATH)).filter(e => !e.startsWith(excludeEnvPrefix));
```

Reply:
done